### PR TITLE
Distribute Inspector Application as a separate npm package

### PR DESCRIPTION
**Pull request by KristinaKoeva**
Release Jenkins builds are not yet updated
Webkit commit SHA would be updated later

### CODE REVIEW

**Comment by KristinaKoeva** [comment]
run ci

File: build/npm/runtime_package.json [hunk 1]
```diff
@@ -22,16 +22,5 @@
...
-  "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-contrib-clean": "^0.5.0",
-    "grunt-contrib-copy": "^0.5.0",
-    "grunt-grunt": "^0.2.2",
-    "grunt-mkdir": "^0.1.2",
-    "grunt-modify-json": "^0.1.1",
-    "grunt-shell": "^0.7.0",
-    "shelljs": "^0.3.0",
-    "ws": "^0.7.1"
```
Comment (jasssonpet) [inline build/npm/runtime_package.json]: Isn't websocket required for the debugger-proxy?

Comment (fealebenpae) [inline build/npm/runtime_package.json]: Well, `npm install -g ws` once on your dev workstation solves this problem.

**Comment by KristinaKoeva** [comment]
run ci

**Comment by KristinaKoeva** [comment]
run ci

**Comment by KristinaKoeva** [comment]
run ci

**Pull request merged**